index.html

```diff
@@ -1663,7 +1686,7 @@ <h3 id="applying">
           end-user, the user agent MUST use the <a>base direction</a> to
           compute directional runs and layout or position text correctly in
           text containing mixed-direction sequences [[!BIDI]]. When the <a>base
-          direction</a> is "<a data-lt="direction auto">auto</a>" the user
+          direction</a> is "<a data-link-for="TextDirectionType">auto</a>" the user
```

Comment:
move `data-link-for="TextDirectionType"` inside ` <dl data-dfn-for="TextDirectionType">`

Reply:
doesnt work, paragraph is not inside <dl>